edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/routing/CloudEndpoint.cs

```diff
@@ -46,6 +50,94 @@ public override void LogUserMetrics(long messageCount, long latencyInMs)
             // TODO - No-op for now
         }
 
+        static class Events
```

Comment:
Let's please keep the static classes Events and Metrics at the bottom of the files.

Reply:
take it, will change.

Reply:
for nested classes, it will sort by access, name, and then static.